app/styles/component/index.scss

```diff
@@ -31,6 +31,7 @@
 @import 'avatar';
 @import 'language-wikis-index';
 @import 'watch-show';
+@import 'affiliate-unit';
```

Comment:
Can this file be lazy loaded?

Reply:
I'm not sure that makes sense if the HTML is injected straight-away. But if it was lazy loaded, how do you propose to do that? I've never seen SCSS lazy loaded as a pattern in any of our apps.